spark3/parquet-compression.choose-types-wisely.scala

```diff
@@ -0,0 +1,191 @@
+// Databricks notebook source
+// Spark: 3.5.1
+// Local: --driver-memory 2G --master 'local[4]' --conf spark.ui.retainedJobs=2
+
+// COMMAND ----------
+
+/*
+This snippet shows how data type for numerical information and compression can affect Spark.
+
+# Symptom
+Storage needs do not match with expectations, for example volume is higher.
+
+# Explanation
+There is difference in type when reading the data and type when writing it, causing a loss of compression perfomance.
+
```

Comment:
Could you please add the `# What to aim for concretely` section, as in other snippets? E.g. choose the right data type according to your needs and taking performance into account. You could reference the example you gave below.